sdk/openai/tools/TestFramework/src/Mocks/MockPage.cs

```diff
@@ -31,7 +31,12 @@ public static MockPage<TValue> FromClientResult(ClientResult result)
     {
         PipelineResponse response = result.GetRawResponse();
         response.BufferContent();
-        return response.Content.ToObjectFromJson<MockPage<TValue>>();
+        MockPage<TValue> mockPage = response.Content.ToObjectFromJson<MockPage<TValue>>() ?? new MockPage<TValue>
```

Comment:
Will `ToObjectFromJson` ever return null here?  Would the null suppression operator be useful, or do we actually need to return a type instance for the case `ToObjectFromJson` returns null?

Reply:
ToObjectFromJson returns T? (https://learn.microsoft.com/en-us/dotnet/api/system.binarydata.toobjectfromjson?view=net-8.0#system-binarydata-toobjectfromjson-1(system-text-json-jsonserializeroptions)) and the implementation returns the result of JsonSerializer.Deserialize with also returns TValue?/object? 

It might not be possible in this case (I'm not sure) but in general it could return null.